google/cloud/bigquery/opentelemetry_tracing.py

```diff
@@ -153,4 +153,12 @@ def _set_job_attributes(job_ref):
     if job_ref.num_child_jobs is not None:
         job_attributes["num_child_jobs"] = job_ref.num_child_jobs
 
+    total_bytes_billed = getattr(job_ref, "total_bytes_billed", None)
```

Comment:
I see that `job_ref` is of class `google.cloud.bigquery.job._AsyncJob`, which doesn't have the two attributes you added (nor does it inherit one). Under what condition will these attribute be present? Should we add them to `_AsyncJob`?